src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -71,7 +72,15 @@ export async function createStaticWebApp(context: IActionContext & Partial<ICrea
     });
 
     await gitPull(nonNullProp(context, 'repo'));
-    await openYAMLConfigFile(context, swaNode);
+
+    const environmentNode = <EnvironmentTreeItem>(await swaNode.loadAllChildren(context)).find(node => {
+        return node instanceof EnvironmentTreeItem && node.inWorkspace;
+    });
+    await environmentNode.loadAllChildren(context);
+    const nodeToReveal: GitHubConfigGroupTreeItem | EnvironmentTreeItem  = environmentNode.gitHubConfigGroupTreeItems.length === 1 ?
```

Comment:
I think from the offline discussion, we want to reveal the `EnvironmentTreeItem`, not the config node.  The idea is just to show them that there is a configuration node.